Backend/app/therapy_sessions/consumers.py

```
@@ -0,0 +1,343 @@
+"""
+WebSocket consumers for therapy sessions.
+Handles secure WebSocket connections for real-time therapy session communication.
+"""
+
+import json
+import uuid
```

Ruff (0.12.2)

`uuid` imported but unused

Remove unused import: `uuid`

(F401)